src/main/webapp/style.css

```diff
@@ -82,13 +82,6 @@
   float: right;
 }
 
-#organization-list {
```

Comment:
this class was not doing anything and removing it allows the see more results button to stay at the bottom of the organizations lists